client/src/app.js

```diff
@@ -3,7 +3,9 @@ dotenv.config();
 import express from 'express';
 import bodyParser from 'body-parser';
 import ejs from "ejs";
+import React from 'react';
 import https from "https";
+import './App.css';
 
 const app = express();
 
```

Comment:
Create a new file named `app.js` in the root directory of the project and move all the code from `client/src/app.js` to that file. And in `client/src/app.js` just render a simple `Hello World` onto the viewport

Reply:
The content of the `app.js` file is server-side code and everything inside the `client` folder is frontend code. We should not put backend code in frontend and must separate any business logic from UI to maintain the separation of concerns